### DISCUSSION TITLE
Maximize Assistant Pane programatically

### DISCUSSION THREAD

**Discussion by mkozjak** [zed-industries/zed#14550] [Feature Requests]
Hi,

I'm using Zed with LLM for programming but also for general questions. I now want to use Zed's Assistant via global OS shortcut. What this means is if Zed is closed, it should get launched and have its Assistant Pane shown and maximized in order to fill all window's available space. Right now I'm using llama.cpp's built-in UI, but I'd like to switch to using Zed instead because the UI is amazing. I have a floating menu widget on macOS that I start with a shortcut and it appears like it's shown on the image below.

Is there a way to run zed on cli and order it to show and maximize the pane somehow?
Thanks!

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/b7bb1117-2263-42b3-90c1-728749ee8a95">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
